### PR TITLE
add `dependabot.yml`

### PR DESCRIPTION
**Pull request by xuwei-k**
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (nicolasstucki) [inline .github/dependabot.yml:6]: I would change this to Tuesday as this aligns better with the start of a new issue supervisor.
```suggestion
      interval: "tuesday"
```

**Review by nicolasstucki — COMMENTED** [review]
Otherwise LGTM

**Review by nicolasstucki — DISMISSED** [review]

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "tuesday"
```
Comment (nicolasstucki) [inline .github/dependabot.yml:6]:
```suggestion
      interval: "weekly"
      day: "tuesday"
```

**Comment by nicolasstucki** [comment]
Sorry for the mess @xuwei-k. Could you squash the commits to cleanup the history?

**Review by SethTisue — APPROVED** [review]

**Pull request merged**